src/Form/AutologoutSettingsForm.php

```diff
@@ -167,36 +199,35 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function validateForm(array &$form, FormStateInterface $form_state) {
-    $input_values = $form_state->getUserInput();
-    $max_timeout = $input_values['autologout_max_timeout'];
+    $max_timeout = $form_state->getValue('autologout_max_timeout');
```

Comment:
Ah, yes I had initially refactored it like then. But then getting all values in array seemed better - which I implemented in `submitForm()`. This missed the treatment. Thanks for the catch.